lib/qless/failure_formatter.rb

```diff
@@ -15,29 +15,25 @@ def initialize
       @replacements[ENV['GEM_HOME']] = '<GEM_HOME>' if ENV.key?('GEM_HOME')
     end
 
+    # lib/qless/job.rb#fail shows us that qless, right down to the Lua scripts,
+    # is set up to expect both a group and a message for a failed job. So we
+    # can't stop storing failed jobs altogether. But, to save on precious RAM,
+    # we can stop recording the message, which is the stack traces that we currently
+    # store.
     def format(job, error, lines_to_remove = caller(2))
       group = "#{job.klass_name}:#{error.class}"
-      message = "#{truncated_message(error)}\n\n" +
-        "#{format_failure_backtrace(error.backtrace, lines_to_remove)}"
+      message = "#{truncated_message(error)}"
       Failure.new(group, message)
     end
 
   private
 
     # TODO: pull this out into a config option.
-    MAX_ERROR_MESSAGE_SIZE = 10_000
+    MAX_ERROR_MESSAGE_SIZE = 100
```

Comment:
i don't remember what the AC was for that ticket but if the point is to keep just the type of error and look to splunk you could probably cut this down even more

Reply:
Yeah, this just keeps the first 100 bytes of the error message. So if the code died because of `raise SomeError.new("Hello")`, "Hello" would get truncated to 100 bytes.

Reply:
OH my bad I misread this as lines